components/definitions/newTab.d.ts

```diff
@@ -196,6 +197,13 @@ declare namespace NewTab {
     wallets: Record<string, number>
   }
 
+  export interface DefaultTopSite {
```

Comment:
Are there any differences from a top site from Chromium and a top site from Super Referral other than the source it comes from? For example, can default top sites be removed, dragged, pinned/un-pinned, and bookmarked as common top sites? If they behave the same they could share the same type interface

Reply:
Currently, it has additional `favicon` and `pinnedIndex` key.
I think `pinnedIndex` is optional now. The main reason why I introduced this new interface was `favicon` key. With `MostVisitedURL`, I got compile failure that `MostVisitedURL` doesn't have `favicon` property. Can we resolve this with `MostVisitedURL`?

Reply:
I see. I'm better with your changes, but default top site makes me think it's a global addition (as we used to have in browser-laptop). If this is something only for SR, maybe better to rename it to `DefaultSuperReferralTopSite` or something similar? To avoid confusion in the future since global default top sites have been on the plans for a while.

Reply:
Yup, changed to `DefaultSuperReferralTopSite`.